package.json

```diff
@@ -29,5 +29,8 @@
   },
   "peerDependencies": {
     "eslint": "^7.1.0"
+  },
+  "dependencies": {
+    "eslint-plugin-unicorn": "^39.0.0"
```

Comment:
I don't know how these eslint plugins work, should this be a dev dep, a peer dep, or a regular one?

Have you tried this against any of the new modules?

Reply:
this has a lot of rules we appear to want such as no use of `Promise.then` (await things only) and stuff like that: https://github.com/sindresorhus/eslint-plugin-unicorn

We don't usually include plugins directly in eslint-config-digitalbazaar. For instance, we allow you to not use the `eslint-plugin-jsdoc` if you so choose, but we might want to include unicorn as if we decide to use any of the rules it provides, we would have to install it anyways. I think, this needs to be a peer Dep for now, but we could open the discussion of including `eslint` and this plugin as `dependencies` in the future as that would cut down on the number of times developers have to install `eslint` and associated plugins in the future. So yeah move this to `peerDependencies`. I might add that the next major release of npm will install `peerDependencies` automatically.

Reply:
I have not tried this on the just recently converted code.

The type of dependencies to use in this case confuses me.  IIRC, at the time I looked around and noticed that the common use case was to have peer dep for eslint itself, and regular dep for things your plugin uses.  That makes installation and instructions easy. (install plugin, add to config)  It's rare to have the setup we have with /vue and /jsdocs where you have to know what other plugins to install.  In our case, it's pretty obvious since the names are there.  In the new case here, I figured it was not obvious, and looking into all the things in -unicorn, I figured maybe we'd pull in some bits into our regular config too.  So... I made it a regular dep.  It should not be a dev dep alone since that implies it's being used to check the plugin itself.

I do not think we should have eslint be a regular dep.  No other plugins do that.

Reply:
AirBnb's config file seems to use `peerDeps` for plugins: https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/package.json

I think we don't need to rock the boat on this PR at least and stick with `peerDeps` for plugins.